### PR TITLE
Compression is set to lzo for faster startup

### PR DESCRIPTION
**Pull request by tunix**

### CODE REVIEW

**Review by igorljubuncic — APPROVED** [review]
Looks fine to me!

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ adopt-info: gradle
 base: core20
 grade: stable
 confinement: classic
+compression: lzo
```
Comment (Lin-Buo-Ren) [inline snap/snapcraft.yaml:7]: Ain't the lzo compression has a snapd version dependency?   We should probably set the `assumes` property to ensure that systems with unsupported snapd version will not try to install the package.

Comment (Lin-Buo-Ren) [reply snap/snapcraft.yaml:7]: According to [Application startup performance - snapd - snapcraft.io](https://forum.snapcraft.io/t/application-startup-performance/17273/13), the first snapd version that supports this feature seems to be 2.47.1.

Comment (om26er) [reply snap/snapcraft.yaml:7]: LZO compression only depends on the kernel support, it's independent of snapd's version.

Comment (Lin-Buo-Ren) [reply snap/snapcraft.yaml:7]: Got it, thanks.

**Comment by Lin-Buo-Ren** [comment]
LGTM.

**Pull request merged**